### PR TITLE
V2.0

### PR DESCRIPTION
**Pull request by Zan1456**
Pictures:
image
image
image
image

### CODE REVIEW

**Comment by TheFantomas** [comment]
Good job thanks we will review it

**Comment by TheFantomas** [comment]
Rewrite functions in english please.

**Comment by TheFantomas** [comment]
image

**Review by TheFantomas — CHANGES_REQUESTED** [review]
Rewrite functions in english please.

File: html/scripts.js [hunk 1]
```diff
@@ -1,137 +1,245 @@
...
+  currentQuestion = 1;
+  faultpoints = 0;
+  maxErrorPoints = 9;
+  correctAnswer = 0;
+  wrongAnswer = 0;
+  passed = false;
+
+  questions = {
+    1: {
+      question: "If you're going 80 km/h, and you're approaching a residential area you must:",
```
Comment (jbdevic) [inline html/scripts.js:11]: I think the questions should be retrieved from an array in the config - would offer more personalization

File: html/scripts.js [hunk 2]
```diff
@@ -1,137 +1,245 @@
...
+  $(".question").fadeIn(1000);
+  setQuestion(drivingSchool.currentQuestion)
+  $(".step-progress__progress").animate({ width: "5%" }, 500 );
+}
+
+function endDrivingSchool() {
+  $(".question").hide(0);
+
+  if(drivingSchool.faultpoints >= drivingSchool.maxErrorPoints) {
+    $(".evaluation-title").html("Oops, that was probably nothing!")
```
Comment (jbdevic) [inline html/scripts.js:161]: make a translation file

**Review by jbdevic — CHANGES_REQUESTED** [review]

**Comment by jbdevic** [comment]
very nice job, I will surely try to improve the ui

File: html/scripts.js (hunk 2 above)
Comment (Zan1456) [reply html/scripts.js:161]: I will implement the changes soon.

File: html/scripts.js (hunk 1 above)
Comment (Zan1456) [reply html/scripts.js:11]: I will implement the changes soon.

**Comment by Zan1456** [comment]
Anything else?

**Comment by Zan1456** [comment]
> sorry I've been busy, we will look at it this week

dump

**Comment by Zan1456** [comment]
I have 2 comments.
1. A file called questions_de.js remained in the html folder.
2. In the fxmanifest.lua file, version should be '2.0' and legacyversion should be '1.9.4'.

**Comment by TheFantomas** [comment]
> I have 2 comments.
> 
> 1. A file called questions_de.js remained in the html folder.
> 2. In the fxmanifest.lua file, version should be '2.0' and legacyversion should be '1.9.4'.

Yes, yes we push it in to main (Whenit is fully functional) this branch was created only for testing.

**Pull request merged**